react/features/base/themes/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './components';
```

Comment:
Please drop the index file. We no longer add it for new features since it helps avoid cycles.